### DISCUSSION TITLE
Is there anyway to import connections/favourites from Panic Transmit into Cyberduck?

### DISCUSSION THREAD

**Discussion by kreativejuices** [iterate-ch/cyberduck#15237] [Q&A]
Hey.

So im moving from Transmit to Cyberduck but have well over 100 FTP connections saved in Transmit. I was wondering if there is any way to migrate them?

Thanks

**Comment by dkocher** [comment]
You should get a prompt to import bookmarks when opening Cyberduck and bookmark files from Panic Transmit 4 or 5 are found on your system [^1].

[^1]: https://docs.cyberduck.io/cyberduck/bookmarks/#from-third-party-applications
